### ISSUE TITLE
UI improvement ideas

### ISSUE THREAD

**Issue by wasamasa** [ashinn/snow-fort#7]
I've created two PRs for a few obvious UI improvements, but I have some ideas for more controversial stuff:

- I only found out that the package listing is sortable by checking the site source. Maybe styling should be changed to display both sorting arrows by default?
- Dunno about you, but I keep clicking the links in the package column and keep getting surprised they prompt me for a download instead of taking me to a detailed package metadata view. I'd rather only have the name there and a dedicated download/source column with the download link there, similar to the documentation.
- The name could lead to a detailed package metadata view. It would contain a prettier version of the information in http://snow-fort.org/pkg/repo. I'm not sure what exactly to include there, besides everything from the list view, but in a row instead of column-oriented form. Candidates:
  - License information
  - Dependencies (with links to dependencies provided by the repo?)
  - Signatures (could be useful for verification)
  - Lesser common fields

Let me know if any of these are worth hacking on.

**Comment by ashinn** [comment]
Yes, this has been on my TODO list and I just haven't gotten around to it.  I also want to include links to older versions, and make the source viewable.

**Comment by wasamasa** [comment]
Ah yes, I've considered these as well, but wasn't sure whether it's even possible to link to older releases or view the sources. An even more outlandish idea: Running a CI system that tries installing a package on every supported platform whenever there's a new release (be it of the package or the Scheme system). The build status could be integrated into the repository page.
